documents/services/records/config.py

```diff
@@ -88,6 +89,7 @@ def components(self):
             CustomFieldsComponent,
             AutorequestComponent,
             WorkflowComponent,
+            NUSLPIDComponent
```

Comment:
Need to add the component to model file as well, see https://github.com/Narodni-repozitar/nr-docs/blob/main/models/documents.yaml#L99